cipher_blacklist.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
I think we'll need some documentation for this... If you ignore elements in this dictionary that are not `*_blacklist`, you could add a pseudo-element before each `*_blacklist`, e.g. `"_comment": "Not implemented"`.

Reply:
Added this kind of comments. It woulg be nice to be able to comment single Cipher Suites, but I guess it's fine like this for now.